client/state/sites/selectors/is-simple-site.js

```diff
@@ -0,0 +1,21 @@
+import { createSelector } from '@automattic/state-utils';
+import { getSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+/**
+ * Returns true if the current site is a simple site
+ *
+ * @param  {object}   state         Global state tree
+ * @returns {?boolean}               Whether the current site is a simple site or not
+ */
+const isSimpleSite = createSelector(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const site = getSite( state, siteId );
+
+		return site && ! site.is_wpcom_atomic;
```

Comment:
I agree here. Also Atomic sites are technically Jetpack sites too, so checking that the site is not a Jetpack site should do better here.

Reply:
True, but I think it's best to be explicit about the expectations. Someone reading the code in the future could then think we're only interested in filtering out Jetpack sites. Seeing `! is_jetpack && ! is_atomic` is a clear message to me. Seeing just `! is_jetpack` could be vague - did the author purposefully leave out the Atomic check knowing they'd be excluded by this as well or did they forget about the Atomic case here (and possibly in other places)?

Plus, the assumption that all Atomic sites are also Jetpack sites could change. Sounds pretty unlikely, but we've seen weirder things happen (and break as a result ;)).

Reply:
Or it may never happen, and Atomic sites will be Jetpack forever. Who knows, maybe we get hit by an electromagnetic pulse and we no longer care about any of this anyway 😄 

I get the point though and have no strong feelings toward any of these. Both have their pros and cons as the discussion shows 😉